### DISCUSSION TITLE
Issue tracker question

### DISCUSSION THREAD

**Discussion by AlexTrotsenko** [LemonAppDev/konsist#1428] [Ask a Question] [answered]
### Wanted end result.

I see, that github issues are not enabled for the project. I did use Discussions to request a feature and got a valuable reply.

But it might be not that clear for everybody in Github what is an expected feedback channel for konsist team.

E.g. in one of the comments I saw the link to the https://lemonappdev.atlassian.net/projects/KON/issues
Shall this issue tracker be used instead ? 

Perhaps it worth to have a section in read me how to report and issue or request a feature.

### What you tried so far.

Reporting a feature request via Discussions

**Comment by igorwojda** [comment, marked as answer]
Thx for the feedback. For feature requests GitHub discussions are way to go (ATM we are using Jira to track work).

I have added section to the readme https://docs.konsist.lemonappdev.com/getting-started/getting-help. I have also updated GitHub readme https://github.com/LemonAppDev/konsist/pull/1429 (develop branch for now)

**Comment by igorwojda** [comment]
-
